url.bs

```diff
@@ -614,11 +612,10 @@ requires context to be distinguished.
 <h3 id=host-parsing>Host parsing</h3>
 
 <p>The <dfn export id=concept-host-parser lt="host parser|host parsing">host parser</dfn> takes a
-string <var>input</var> with an optional boolean <var>isNotSpecial</var>, and then runs these steps:
+string <var>input</var> with an optional boolean <var>isNotSpecial</var> (default false), and then
```

Comment:
Needs dfn (and since this is internal you can also update the call sites)

Reply:
See above.